opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/presenter/AllClientsRegisterPresenter.java

```diff
@@ -0,0 +1,32 @@
+package org.smartregister.chw.hf.presenter;
+
+import org.apache.commons.lang3.tuple.Triple;
+import org.smartregister.opd.contract.OpdRegisterActivityContract;
+import org.smartregister.opd.presenter.BaseOpdRegisterActivityPresenter;
+
+public class AllClientsRegisterPresenter extends BaseOpdRegisterActivityPresenter {
+
+    public AllClientsRegisterPresenter(OpdRegisterActivityContract.View view, OpdRegisterActivityContract.Model model) {
+        super(view, model);
+    }
+
+    @Override
+    public void onNoUniqueId() {
+        //Overridden
+    }
+
+    @Override
+    public void onUniqueIdFetched(Triple<String, String, String> triple, String s) {
+        //Overridden
```

Comment:
Is it necessary to override this. If so give a descriptive message as to why they are not used but implemented